paper.bib

```diff
@@ -0,0 +1,107 @@
+@article{frontier,
+    author = {Barb Helland},
+    title = {At the Frontier: DOE Supercomputing Launches the Exascale Era},
+    year = {2022},
+    publisher = {United States Department of Energy Office of Science},
+    url = {https://www.energy.gov/science/articles/frontier-doe-supercomputing-launches-exascale-era}
+}
+
```

Comment:
This is not referenced anymore, as the respective paragraph was deleted. I'm not sure if it fits into the new structure. If not, the entire entry can be removed.
```suggestion
```